src/providers/ad/ad_id.c

```diff
@@ -1188,6 +1189,23 @@ ad_get_account_domain_send(TALLOC_CTX *mem_ctx,
         }
     }
 
+    /* SID lookup does not require communication with backend */
+    if (state->entry_type == BE_REQ_BY_SECID) {
+        domain = find_domain_by_sid(params->domain, data->filter_value);
+        if (domain == NULL) {
+            DEBUG(SSSDBG_TRACE_INTERNAL,
```

Comment:
I wonder if this should be `SSSDBG_OP_FAILURE` or `SSSDBG_IMPORTANT_INFO`...

Reply:
I think that this can happen normally when we search in multi-domain environment therefore I think that SSSDBG_TRACE_INTERNAL is good choice here.

Reply:
`find_domain_by_sid()` iterates all known domains.
Fail of this functions means SID is "totally" unknown, right?
What can be a source of "unknown" SID? Can this be a sign of some "misconfig"? Doesn't admin need to be aware?

Reply:
Not exactly - it means that this SID is not known to this particular back-end instance. Theoretically we can have more back-ends with SIDs running.

I do not think that this is important to the admin, only for troubleshooting.

Reply:
Thanks for the explanation.